tcpdump.c

```diff
@@ -688,6 +754,7 @@ static const struct option longopts[] = {
 #ifdef HAVE_PCAP_SETDIRECTION
 	{ "direction", required_argument, NULL, 'Q' },
 #endif
+  { "max-runtime", required_argument, NULL, 'R', },
```

Comment:
Please use the same indentation as other code.

Note: tab stops are set every *eight* spaces, not every *four* spaces, this having originally been written on UN\*X. :-)  If you're reading it with a program with tab stops set every four spaces, it won't display correctly.

Reply:
Will clean up when I rebase :)